modules/new_serverpod_auth/serverpod_auth_core/serverpod_auth_core_server/lib/src/jwt/endpoints/jwt_tokens_endpoint.dart

```diff
@@ -11,7 +11,7 @@ abstract class RefreshJwtTokensEndpoint extends Endpoint {
   /// [AuthConfig.getTokenManager] method.
   ///
   /// If a different token manager should be used, override this property.
-  final AuthenticationTokens authenticationTokens =
+  AuthenticationTokens get authenticationTokens =>
       AuthConfig.getTokenManager<AuthenticationTokensTokenManager>()
           .authenticationTokens;
```

Comment:
Would it make sense to use this pattern:
```dart
late final authenticationTokens =
      AuthConfig.getTokenManager<AuthenticationTokensTokenManager>()
          .authenticationTokens;
```
So that it is constructed late, but only on first access?